cea/demand/thermal_loads.py

```diff
@@ -974,10 +974,7 @@ def calc_prop_rc_model(self, occupancy, architecture, thermal_properties, geomet
         df['Af'] = df['GFA_m2'] * df['Hs']  # conditioned area - areas not heated
         df['Aef'] = df['GFA_m2'] * df['Es']  # conditioned area only those for electricity
 
-        # FIXME: why are we hard-coding 'Cm' here? and can we do without it?
-        if 'Cm' in self.get_overrides_columns():
-            df['Cm'] = self._overrides['Cm']
-        else:
+        if 'Cm' not in self.get_overrides_columns():
```

Comment:
does this still work? I'm not sure... what happens if `'cm' is in self.get_overrides_columns()`?